app/views/layouts/application.html.slim

```diff
@@ -9,7 +9,8 @@ html
 
     = favicon_link_tag
 
-    meta name="apple-itunes-app" content="app-id=#{Rails.application.secrets.mobile_app_id["ios"]}"
+    /Removed due to the implementation of a custom smart banner
+    /meta name="apple-itunes-app" content="app-id=#{Rails.application.secrets.mobile_app_id["ios"]}"
```

Comment:
Do you think this is the best approach? Maybe we should keep the native banner on iOS, as the users may be used it.